Fix code snippets in RUM > iOS > Advanced Configuration (#27617)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/ios/advanced_configuration.md

{{% /tab %}}
{{< /tabs >}}

> For more details and available options, [see `RUMMonitorProtocol` on GitHub][4].

Reviewer: ```suggestion
For more details and available options, [see `RUMMonitorProtocol` in GitHub][4].
```

{{% /tab %}}
{{< /tabs >}}

**Note**: When using `.startAction(type:name:)` and `.stopAction(type:)`, the action `type` must be the same. This is necessary for the RUM iOS SDK to match an action start with its completion.

> For more details and available options, [see `RUMMonitorProtocol` on GitHub][4].

Reviewer: ```suggestion
For more details and available options, [see `RUMMonitorProtocol` in GitHub][4].
```

**Note**: The `String` used for `resourceKey` in both calls must be unique for the resource you are calling. This is necessary for the RUM iOS SDK to match a resource's start with its completion.

> For more details and available options, [see `RUMMonitorProtocol` on GitHub][4].

Reviewer: ```suggestion
For more details and available options, [see `RUMMonitorProtocol` in GitHub][4].
```

{{% /tab %}}
{{< /tabs >}}

> For more details and available options, [see `RUMMonitorProtocol` on GitHub][4] and [Error Attributes documentation][5].

Reviewer: ```suggestion
For more details and available options, [see `RUMMonitorProtocol` in GitHub][4] and the [Error Attributes documentation][5].
```

* To update the value, use `RUMMonitor.shared().addAttribute(forKey: "<KEY>", value: "<UPDATED_VALUE>")`.
* To remove the key, use `RUMMonitor.shared().removeAttribute(forKey: "<KEY_TO_REMOVE>")`.

> For better performance in bulk operations (modifying multiple attributes at once) use `.addAttributes(_:)` and `.addAttributes(_:)`.

Reviewer 2: `.addAttributes(_:)` and `.removeAttributes(forKeys:)`

Author: Good catch 👍 
```suggestion
For better performance in bulk operations (modifying multiple attributes at once) use `.addAttributes(_:)` and `. removeAttributes(forKeys:)`.
```

> For better performance in bulk operations (modifying multiple attributes at once) use `.addAttributes(_:)` and `.addAttributes(_:)`.

Reviewer: ```suggestion
For better performance in bulk operations (modifying multiple attributes at once), use `.addAttributes(_:)` and `.addAttributes(_:)`.
```

> For better performance in bulk operations (modifying multiple attributes at once), use `.addAttributes(_:)` and `. removeAttributes(forKeys:)`.

Author: ```suggestion
For better performance in bulk operations (modifying multiple attributes at once), use `.addAttributes(_:)` and `.removeAttributes(forKeys:)`.
```

You have the option of deleting all unsent data stored by the SDK with the `Datadog.clearAllData()` API.

```swift
   public static func clearAllData(in core: DatadogCoreProtocol = CoreRegistry.default) {
       let core = core as? DatadogCore
       core?.clearAllData()
   }
import DatadogCore

Datadog.clearAllData()
```

### Stop data collection

You can use the `StopInstance` API to stop a named SDK instance (or the default instance if the name is null) from collecting and uploading data further.
You can use the `Datadog.stopInstance()` API to stop a named SDK instance (or the default instance if the name is `nil`) from collecting and uploading data further.

```swift
   public static func stopInstance(named instanceName: String = CoreRegistry.defaultInstanceName) {
       let core = CoreRegistry.unregisterInstance(named: instanceName) as? DatadogCore
       core?.stop()
   }
```
import DatadogCore

### Set remote log threshold

You can define the minimum log level (priority) at which to send events to Datadog in a logger instance. If the log's priority is below the one you set in this threshold, the log does not get sent. The default value is `debug` (send all logs).

> 

Author: 💬 This doesn't belong to RUM, it touches on [iOS Logs](https://docs.datadoghq.com/logs/log_collection/ios/?tab=swiftpackagemanagerspm) but the code snippet is very off.
